src/components/Authentication/Register.jsx

```diff
@@ -0,0 +1,199 @@
+/* eslint-disable no-console */
+/* eslint-disable react/no-children-prop */
+import React, { useState } from 'react';
+import { string } from 'prop-types';
+import { Link, useNavigate } from 'react-router-dom';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Link' is defined but never used.